src/libraries/System.Private.Runtime.InteropServices.JavaScript/src/System/Runtime/InteropServices/JavaScript/Private/JavaScriptMarshalImpl.Bind.cs

```diff
@@ -12,10 +12,10 @@ public static Action<T1> BindJSActionByName<T1>(string functionName)
         {
             const int argsCount = 1;
             var extraBufferLength = 0;
-            var excSig = GetExceptionMeta();
-            var resSig = GetVoidMeta();
-            var arg1Sig = GetArgumentMeta<T1>(ref extraBufferLength);
-            var signature = stackalloc JavaScriptMarshallerArgSig[] { excSig, resSig, arg1Sig };
+            var excSig = GetExceptionSignature();
+            var resSig = GetVoidSignature();
+            var arg1Sig = GetArgumentSignature<T1>(ref extraBufferLength);
+            var signature = stackalloc JavaScriptMarshallerArgSignature[] { excSig, resSig, arg1Sig };
```

Comment:
you could probably add those signatures inline so they don't have to go into temporary locals, i.e. ``` { GetExceptionSignature(), ```

Reply:
done